frontend/src/views/setting/panel/api-interface/index.vue

```diff
@@ -171,7 +169,7 @@ const resetApiKey = async () => {
         });
 };
 
-const onBind = async (formEl: FormInstance | undefined) => {
+const onSave = async (formEl: FormInstance | undefined) => {
     if (!formEl) return;
     formEl.validate(async (valid) => {
         if (!valid) return;
```

Comment:
There are no syntax errors, however I would recommend fixing the typo of "commons.button.cancel" to "commons.button.close". This is just a minor point.

The function names seem fine but it's generally recommended to follow consistent naming conventions. Also using variable names like `$t`, `apiURL` and `panelURL` with underscores may not always be universally understood so they should either stay lowercase without underscore (e.g, t for translation).
No other notable issues found during review. The overall structure looks pretty good!